Templates/releaseTemplate.yml

```diff
@@ -0,0 +1,186 @@
+parameters:
+  - name: stageName
+    type: string
+    default: none
+  - name: stageDisplayname
+    type: string
+    default: none
+  - name: runStage
+    type: boolean
+    default: true
+  # - name: dependsOn
+  #   default: []
+  - name: jobName
+    type: string
+    default: none
+  - name: jobDisplayname
+    type: string
+    default: none
+  - name: vmImage
+    type: string
+    default: windows-2022
+  - name: jobcondition
+    type: string
+    default: ''
+  - name: jobDemand
+    type: string
+    default: ''
+
+    stages:
+      - stage: ${{ parameters.stageName }}
+        condition: ${{ parameters.runStage }}
+        dependsOn: []
+        displayName: ${{ parameters.stageDisplayname }}
+        jobs:
+        - job: ${{ parameters.jobName }}
+          timeoutInMinutes: 0
+          cancelTimeoutInMinutes: 1
+          condition: ${{ parameters. jobcondition}}
+          displayName: ${{ parameters.jobDisplayname }}
+
+          variables:
+            TemplateS3Namespace: '/image-cd-pipeline'
+
+          pool:
+            vmImage: ${{ parameters.vmImage }}
+            demands: ${{ parameters.jobDemand }}
+
+          steps:
+          - download: _BuildCloudAccountIdArtefacts
+            displayName: Download artifacts from Build Cloud Account Id Artefacts
+
+          - download: _BuildImageReleaseArtefacts
+            displayName: Download artifacts from Build Image Release Artefacts 2.0
+
+          - checkout: self
+
+          - checkout: _LansaAWSTemplates
+            displayName: Checkout nkhl-accolite/aws-templates-nkhl
+
+          - checkout: _Robe070Cookbooks
+            displayName: Checkout nkhl-accolite/cookbooks
+
+          - task: PowerShell@2
+            displayName: 'Artifact Check : Set Gate Variable'
+            inputs:
+              targetType: filePath
+              filePath: './$(System.DefaultWorkingDirectory)/_robe070_cookbooks/scripts/azure_set_gate_variable_cloud_account_id.ps1'
+              arguments: '-Version "$(System.JobDisplayName)" -osName "Windows Server 2019"'
+              
+          #Your build pipeline references an undefined variable named ‘Gate.IsEnabled’. Create or edit the build pipeline for this YAML file, define the variable on the Variables tab. See https://go.microsoft.com/fwlink/?linkid=865972
```

Comment:
There is some messy code in here...

Needs fixing.

Reply:
removed this file from the directory.